src/qt/clientmodel.cpp

```diff
@@ -142,7 +142,10 @@ OptionsModel *ClientModel::getOptionsModel() { return optionsModel; }
 PeerTableModel *ClientModel::getPeerTableModel() { return peerTableModel; }
 BanTableModel *ClientModel::getBanTableModel() { return banTableModel; }
 QString ClientModel::formatFullVersion() const { return QString::fromStdString(FormatFullVersion()); }
-QString ClientModel::formatSubVersion() const { return QString::fromStdString(strSubVersion); }
```

Comment:
I think with this change strSubVersion can be removed from global state (net.h/.cpp) and just made a local variable in init.cpp, it isn’t used anywhere else.

Reply:
fixed